cmd/export_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package cmd
+
+import (
+	"github.com/juju/cmd"
+
+	"github.com/juju/juju-restore/core"
+)
+
+func NewRestoreCommandForTest(connectF func() (core.Database, func(), error)) cmd.Command {
```

Comment:
I'd prefer to avoid export_test files as much as possible, what about just making NewRestoreCommand accept a connect function?